src/mono/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilderInstantiation.cs

```diff
@@ -558,7 +558,7 @@ abstract class TypeBuilderInstantiation : TypeInfo
 	{
 		internal static Type MakeGenericType (Type type, Type[] typeArguments)
 		{
-			throw new NotSupportedException("User types are not supported under full aot");
+			throw new NotSupportedException(SR.NotSupported_UserTypesNotSupportedUnderFullAot);
```

Comment:
```suggestion
			throw new NotSupportedException(SR.PlatformNotSupported_ReflectionEmit);
```
Similar existing error message